packages/frontend/src/features/metricsCatalog/components/MetricPeekModal.tsx

```diff
@@ -121,18 +122,37 @@ export const MetricPeekModal: FC<Props> = ({ opened, onClose }) => {
             };
         }, [metricResultsQuery.data?.fields]);
 
-    useEffect(() => {
-        const timeDimension =
-            timeDimensionOverride ?? metricQuery.data?.timeDimension;
-
-        if (!timeDimension) return;
+    useEffect(
+        function setInitialDateRange() {
```

Comment:
its unusual to see `function` keyword instead of arrow functions. im just curious to hear your take on this. is it for error reporting to have named functions?

Reply:
easier for me to know what the useEffect does 😓 I saw this pattern once and feel like it makes it super clear - to me at least!

Reply:
so it acts as a description of what useEffect does. I like it!

Reply:
exactly!